### PR TITLE
2.0.0 Bump on Next (Client Packages)

### PR DESCRIPTION
**Pull request by ssimic2**
Bumping client packages to 2.0.0.

@fluidframework/build-common:     0.24.0 (unchanged)
@fluidframework/eslint-config-fluid:  0.29.1000 (unchanged)
@fluidframework/common-definitions:  0.21.1000 (unchanged)
@fluidframework/common-utils:  0.33.1000 (unchanged
@fluidframework/protocol-definitions: 0.1029.1000 (unchanged)
Azure:  0.60.1000 (unchanged)
Server: 0.1037.1000 (unchanged)
Client:      1.1.0 -> 2.0.0     
@fluid-tools/benchmark:     0.41.0 (unchanged)
tinylicious:      0.4.0 (unchanged)
@fluidframework/azure-local-service:      0.1.0 (unchanged)

### CODE REVIEW

File: packages/framework/tinylicious-client/package.json [hunk 1]
```diff
@@ -41,20 +41,20 @@
   "dependencies": {
```
Comment (skylerjokiel) [inline packages/framework/tinylicious-client/package.json:41]: Please update the peerDependencies "fluid-framework": "^2.0.0"

I can't comment directly on the line in github.

**Review by skylerjokiel — APPROVED** [review]
One place that needs updating.

**Comment by ssimic2** [comment]
> One place that needs updating.

Good catch. There are few more looks like.

**Comment by ssimic2** [comment]
> > One place that needs updating.
> 
> Good catch. There are few more looks like.

OK. this may be it.

**Comment by markfields** [comment]
@ssimic2 Looks like we need a follow-up PR that updates the type validation stuff.  The blob at the bottom of the package.json's still says 1.0.0.  cc @anthony-murphy

**Pull request merged**